### PR TITLE
MES-4898 - Added View Test Result for Home Tests

### PR DESCRIPTION
**Pull request by TomBillingtonUK**
## Description
- Added the placeholder page for View test Result

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by faggc — APPROVED** [review]

File: src/pages/view-test-result/cat-home-test/__tests__/view-test-result.cat-home-test.page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+        component.handleLoadingUI(false);
+
+        expect(component.isLoading).toEqual(false);
+        expect(component.loadingSpinner).toBeNull();
+      });
+    });
+    describe('getTestDetails', () => {
+      it('should correctly generate the data', () => {
+        // TODO - Cat Home , use correct mock
```
Comment (rstepnowski) [inline src/pages/view-test-result/cat-home-test/__tests__/view-test-result.cat-home-test.page.spec.ts:112]: Soo.... do we need a proper mock here or not ? :D // + other instance of the same todo below

File: src/pages/view-test-result/cat-home-test/components/debrief-card/__tests__/debrief-card.spec.ts [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+import { FaultsDataRowComponent } from '../../../../components/faults-data-row/faults-data-row';
+import { VehicleChecksDataRowComponent } from '../../../../components/vehicle-checks-data-row/vehicle-checks-data-row';
+import { FaultSummaryProvider } from '../../../../../../providers/fault-summary/fault-summary';
+import { FaultCountProvider } from '../../../../../../providers/fault-count/fault-count';
+import {
+  DataRowListItem,
+  TestRequirementsLabels,
+  ViewTestResultLabels,
+} from '../../../../components/data-row-with-list/data-list-with-row.model';
+// TODO - Cat HOME , use correct labels
```
Comment (rstepnowski) [inline src/pages/view-test-result/cat-home-test/components/debrief-card/__tests__/debrief-card.spec.ts:28]: Another todo with unknown status? :>

File: src/pages/view-test-result/cat-home-test/components/debrief-card/__tests__/debrief-card.spec.ts [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+  beforeEach(async(() => {
+    fixture = TestBed.createComponent(DebriefCardComponent);
+    component = fixture.componentInstance;
+    faultSummaryProvider = TestBed.get(FaultSummaryProvider);
+    faultCountProvider = TestBed.get(FaultCountProvider);
+  }));
+
+  describe('Class', () => {
+    describe('getTestRequirements', () => {
+      // TODO - Cat HOME - needs to be fixed
```
Comment (rstepnowski) [inline src/pages/view-test-result/cat-home-test/components/debrief-card/__tests__/debrief-card.spec.ts:74]: Are we planning to fix those?

**Review by rstepnowski — APPROVED** [review]
Looks fine if you ignore the outstanding todos in the code  ;)

**Review by appleabs — APPROVED** [review]

**Pull request merged**